test/cypress/integration/widgets/plugins_num_spec.ts

```diff
@@ -0,0 +1,40 @@
+describe('Number of Plugins widget', () => {
+    const widgetId = 'pluginsNum';
+
+    function clickOnWidget() {
+        cy.get('.pluginsNumWidget .statistic').click();
```

Comment:
Maybe:
```suggestion
        cy.getWidget(widgetId).find('.statistic').click();
```
would be a bit cleaner?

Reply:
Done!